amarachukwu-agbo/random-number-generator
# #165145150 Create API endpoint to generate random numbers

#### What does this PR do?
Sets up jest for testing

#### Description of Task to be completed?
- Create route to generate new numbers
- Create controller for generating phone numbers
- Modify babel config file for testing
- Modify jest config file to collect coverage
- Write tests for endpoint

#### How should this be manually tested?
- Clone the repo and install application dependencies using `yarn`. 
- Run command  `yarn server:start-dev`.
- Using Postman, visit the url `http://localhost:3000/api/v1/numbers`, you should see a list of 1000 generated phone numbers.
- Run `yarn test` to test the feature

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145150](https://www.pivotaltracker.com/story/show/165145150)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
